app/src/components/Statistics/Statistics.styles.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+
+export const StyledWrapper = styled.section`
+  width: 80rem;
+  height: 180vh;
```

Comment:
This is not acceptable :( Please refactor that to use normal values

Reply:
sorry, those 180vh was just for testing.